about.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles.css" rel="stylesheet"><title>About The Platform</title>
+        <style>
+            @import url('http://fonts.cdnfonts.com/css/cocogoose');
+        </style>
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+        </style>
+    </head>
+    <body>
+        <header class="navbar-container">
+            <div class="navbar-cont">
+                <nav class="nav-icon-box">
+                    <ul class="nav-icons">
+                        <li><a href="#"><img src="images/Linkedinicon.png" alt=""></a></li>
+                        <li><a href="#"><img src="images/twitter.png" alt=""></a></li>
+                        <li><a href="#">English</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Logout</a></li>
+                    </ul>
+                </nav>
+                <nav class="navbar">
+                    <a href="homepage.html" class="logo"><img src="images/The-Platform-Logo.png" alt="The plaform logo"></a>
```

Comment:
- [ ] This element is not appearing in your mobile view, for day 1 it is only required to have both pages adapted to mobile (which you did) but it is also a requirement to have navigation from the home page to the about and vice-versa, currently, if you are already in the about page, you are unable to go back to home because this element disappears in the mobile view, please make this change in order to comply with the project requirements.